.github/workflows/ci.yml

```diff
@@ -2,19 +2,33 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+    branches:
+      - master
+      - 'v*' # older version branches
+    tags:
+      - '*'
+  pull_request: {}
+  schedule:
+    - cron: '0 6 * * 0' # weekly, on sundays
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
+  test:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node: ['10', '12', '14', '15']
+        os: [ubuntu-latest, macOS-latest, windows-latest]
```

Comment:
FWIW, I don't think this provides any value. All of our tests run in the browser which is the same across all of these matrix setups...